src/api/cvc4cpp.cpp

```diff
@@ -2555,6 +2555,16 @@ Term Solver::mkBitVector(uint32_t size, std::string& s, uint32_t base) const
   CVC4_API_SOLVER_TRY_CATCH_END;
 }
 
+Term Solver::mkConstArray(Sort sort, Term val) const
+{
+  CVC4_API_SOLVER_TRY_CATCH_BEGIN;
+  CVC4_API_CHECK(sort.isArray())  << "Not an array sort.";
+  CVC4_API_CHECK(sort.getArrayElementSort() == val.getSort())  << "Value does not match element sort.";
```

Comment:
Isn't here a check missing? `val` may not be null, right? (macro `CVC4_API_ARG_CHECK_NOT_NULL`)

Reply:
Great catch, thanks! Just added that.